examples/microphone_streaming_example.py

```diff
@@ -99,7 +99,7 @@ def generator(self):
     # Uses try method to allow users to manually close the stream
     try:
         # Starts the server connection and thread sending microphone audio
-        response_gen = streamclient.start(stream.generator())
+        response_gen = streamclient.start(stream.generator(), filter_profanity=True)
```

Comment:
Any reason why we have the example filter_profanity set to true here?

Reply:
I'm just illustrating the usage as per the AC of the issue.